workspace_host/interface.js

```diff
@@ -64,6 +65,8 @@ app.post('/', function(req, res) {
         resetSequence(workspace_id, res);
     } else if (action == 'destroy') {
         destroySequence(workspace_id, res);
+    } else if (action == 'grade') {
+        gradeSequence(workspace_id, res);
```

Comment:
At some point it would be great to refactor this `post('/')` handler to be more like the PL ones with RESTful endpoints. For example, we should have routes like:

* `post('/workspaces/:workspace_id/init')` -- rename this to 'start'?
* `get('/workspaces/:workspace_id/getGradingFiles')`

Reply:
Added TODO note